test/extensions/filters/http/wasm/test_data/body_cpp.cc

```diff
@@ -34,27 +34,61 @@ FilterHeadersStatus ExampleContext::onResponseHeaders(uint32_t) {
   return FilterHeadersStatus::Continue;
 }
 
-void ExampleContext::logBody(BufferType bt) {
-  size_t bufferedSize;
+void ExampleContext::logBody(BufferType type) {
```

Comment:
type->buffer_type for consistency or move them all to "type"

Reply:
They are all `type` here.